theano/sandbox/cuda/dnn.py

```diff
@@ -2246,9 +2247,13 @@ def local_dnn_conv_inplace(node):
         inputs = list(node.inputs)
         dest = inputs[2]
         if (dest.owner and
-                isinstance(dest.owner.op, GpuAllocEmpty) and
+                type(dest.owner.op) is GpuAllocEmpty and
```

Comment:
`GpuAlloc` inherits from `GpuAllocEmpty`, so the previous check would have matched `GpuAlloc` nodes as well.